packages/cadl-ranch-specs/http/enums/fixed/main.cadl

```diff
@@ -0,0 +1,40 @@
+import "@cadl-lang/rest";
+import "@azure-tools/cadl-azure-core";
+import "@azure-tools/cadl-ranch-expect";
+
+using Cadl.Http;
+
+@scenarioService("/fixed-enums")
+namespace FixedEnums;
```

Comment:
I would put both those scenarios into the same file so with the following structure


```
namespace Enums;

namespace Extensible {
 ...
}
namespace Fixed {}
```

or at least have the namespace called like that so in the dashboard it is collapsed under `Enums`